src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -92,7 +92,7 @@ private void rearrangeSectionWithDownStation(Section section) {
                 .findFirst()
                 .ifPresent(it -> {
                     // 신규 구간의 상행역과 기존 구간의 상행역에 대한 구간을 추가한다.
-                    sections.add(new Section(section.getLine(), it.getUpStation(), section.getUpStation(), it.getDistance() - section.getDistance()));
+                    sections.add(new Section(section.getLine(), it.getUpStation(), section.getUpStation(), it.getDistance() - section.getDistance(), section.getDistance()));
```

Comment:
getDistance 대신 getDuration이 아닐까요?
```suggestion
                    sections.add(new Section(section.getLine(), it.getUpStation(), section.getUpStation(), it.getDistance() - section.getDistance(), section.getDuration()));
```